samples/snippets/fsharp/VS_Snippets_CLR_System/system.Action/fs/Delegate.fs

```diff
@@ -0,0 +1,21 @@
+module Delegate
+
+// <Snippet1>
+open System.Windows.Forms
+
+type ShowValue = delegate of unit -> unit
+
+type Name(name) =
+    let instanceName = name
```

Comment:
Same comment here, I think we can be a bit more slim, plus it helps highlight and show off the scoping available to F# classes!